src/controllers/session/login/login.scss

```diff
@@ -24,3 +24,26 @@
         width: fit-content;
     }
 }
+
+.inputContainer {
+    position: relative;
+}
+
+.passwordToggle {
+    position: absolute;
+    width: 30px;
+    height: 30px;
+    top: calc(50% - 0.2em);
+    right: 10px;
```

Comment:
We should use `em` because fonts can be changed by theme and `px` won't match.
Can we use some existing class so that the button can also update by theme?
Or even use `paper-emby-button` here?